src/pages/test-site3.js

```diff
@@ -0,0 +1,57 @@
+import {useState} from 'react';
```

Comment:
Why is this file named 'test-site3' ?
In a PR should be only production ready code.
This also applies to the other files.

Reply:
correct